app/services/projects/bulk_tagger.rb

```diff
@@ -0,0 +1,37 @@
+module Projects
+  class BulkTagger
+    def initialize(content_items:, taxons:)
+      @content_ids = content_items
+      @taxon_content_ids = taxons
+    end
+
+    def commit
+      @content_ids.each do |id|
```

Comment:
could this use some error-handling, say for instance `content_item_id_lookup` fails to return something sensible.